---
permalink: /
title: "Shiyu Jiang"
excerpt: ""
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

{% if site.google_scholar_stats_use_cdn %}
{% assign gsDataBaseUrl = "https://cdn.jsdelivr.net/gh/" | append: site.repository | append: "@" %}
{% else %}
{% assign gsDataBaseUrl = "https://raw.githubusercontent.com/" | append: site.repository | append: "/" %}
{% endif %}
{% assign url = gsDataBaseUrl | append: "google-scholar-stats/gs_data_shieldsio.json" %}

<span class='anchor' id='about-me'></span>

# üó® About Me
I am a full-time Lab Specialist at the Center for Public Health Genomics in University of Virginia School of Medicine
advised by [Prof. Chongzhi Zang](https://zanglab.github.io/index.htm).

Previously, I received my Master's degree in Computer Science from Johns Hopkins University and 
Bachelor's degree in Computer Science from Wenzhou-Kean University. 
I did my master thesis under Prof. Kimia Ghobadi at Johns Hopkins on multi-agent simulation.
I was fortunate to work with Prof. Aloysius Wong's group where I first stepped into the field of bioinformatics. 
In addition, I developed a deep learning model for RNA-ligand affinity prediction with Prof. Yanjun Li at UFL 
and later participated in developing a single-cell foundation model in Prof. Xiaojie Qiu's Lab at Stanford.

I mainly work on the intersection of machine learning and biology in a broad scope. I am interested in foundation 
models and LLMs for science including single-cell, RNA sequence, and protein, and how to use them to discover and 
interpret the biological mechanism.

My research interests focus on:
- Single-cell Multi-omics
- Deep learning for Cancer Biology and Chemical Biology
- AI for Drug Discovery

[//]: # (- Artificial Life)

# üìñ Educations

[//]: # (- *2024.08 - Present*, Doctor of Philosophy, Computational Biology.)
- *2022.08 - 2023.12*, Master of Science in Engineering, Computer Science. Johns Hopkins University. *Baltimore, MD* 
- *2018.08 - 2022.05*, Bachelor of Science, Computer Science. Wenzhou-Kean University. *Wenzhou, China*

# üì∞ News

[//]: # (- *2024.08*: )
[//]: # (- *2024.06*: The follow-up work of building an emotion model for [Koudou]&#40;https://github.com/caranha/Koudou&#41; has been accepted by ALIFE 2024. )
[//]: # (- *2024.03*: [ContraBind]&#40;&#41;, a contrastive learning based model for RNA-small molecules binding affinity prediction has been published as a preprint.)
[//]: # (- *2024.03*: [REFEREE]&#40;&#41;, a federated learning based single-cell foundation model has been published as a preprint.)
[//]: # (- *2024.03*: SICER2, a redesigned ChIP-seq broad peak calling tool, has been updated &#40;check the [Documentation]&#40;https://zanglab.github.io/SICER2/&#41;&#41;.)
[//]: # (- *2023.07*: The work to build based on a multi-agent model for pandemic simulation, [Koudou]&#40;https://github.com/caranha/Koudou&#41;, has been accepted by ALIFE 2023.)

# üìù Publications

[//]: # (<div class='paper-box'><div class='paper-box-image'><div><div class="badge">bioRxiv</div><img src='images/nanoplastics.png' alt="sym" width="75%"></div></div>)

[//]: # (<div class='paper-box-text' markdown="1">)

[//]: # ()
[//]: # ([....]&#40;&#41;)

[//]: # ()
[//]: # (Haoxin Ye, **Shiyu Jiang**, ... and Tianxin Yang<sup>&ast;</sup>. *Preprint*, 2024.)

[//]: # ()
[//]: # ([Code]&#40;https://github.com/Haoxin01/ML_Nanoplastics_MPNs_SERS&#41;)

[//]: # ()
[//]: # (</div>)

[//]: # (</div>)


[//]: # (<div class='paper-box'><div class='paper-box-image'><div><div class="badge">bioRxiv</div><img src='images/complex.png' alt="sym" width="75%"></div></div>)

[//]: # (<div class='paper-box-text' markdown="1">)

[//]: # ()
[//]: # ([Contrastive learning predicts binding affinity between RNA sequences and ligands through adapting foundation models]&#40;&#41;)

[//]: # ()
[//]: # (**Shiyu Jiang**, ... and Yanjun Li<sup>&ast;</sup>. *Preprint*, 2024.)

[//]: # ()
[//]: # ([Code]&#40;&#41;)

[//]: # ()
[//]: # (</div>)

[//]: # (</div>)


[//]: # (<div class='paper-box'><div class='paper-box-image'><div><div class="badge">bioRxiv</div><img src='images/sctab.png' alt="sym" width="75%"></div></div>)

[//]: # (<div class='paper-box-text' markdown="1">)

[//]: # ()
[//]: # ([REFEREE: A privacy-preserving federated foundation model for single-cell analysis via tabular learning]&#40;&#41;)

[//]: # ()
[//]: # (Jiayuan Ding<sup>‚Ä†</sup>, Jianhui Lin<sup>‚Ä†</sup>, **Shiyu Jiang<sup>‚Ä†</sup>**, Yixin Wang, Yuzhen Mao, Lingfeng Shen, Jiliang Tang<sup>&ast;</sup>, Min Li<sup>&ast;</sup>, and Xiaojie Qiu<sup>&ast;</sup>. *Preprint*, 2024.)

[//]: # ()
[//]: # ([Code]&#40;https://github.com/OmicsML/scTab&#41;)

[//]: # ()
[//]: # (</div>)

[//]: # (</div>)


[//]: # (<div class='paper-box'><div class='paper-box-image'><div><div class="badge">Bioinformatics</div><img src='images/sicer2.png' alt="sym" width="75%"></div></div>)

[//]: # (<div class='paper-box-text' markdown="1">)

[//]: # ()
[//]: # ([SICER 2.0: a redesigned ChIP-seq broad peak identified tool with enhanced FDR control and accelerated performance]&#40;&#41;)

[//]: # ()
[//]: # (**Shiyu Jiang**, Jin Yong Yoo, Yiren Wang, and Chongzhi Zang<sup>&ast;</sup>. *Bioinformatics*, 2024.)

[//]: # ()
[//]: # ([Doc]&#40;https://zanglab.github.io/SICER2/&#41; | [Code]&#40;https://github.com/zanglab/SICER2&#41;)

[//]: # ()
[//]: # (</div>)

[//]: # (</div>)


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">ALIFE 2023</div><img src='images/covid_sim.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Simulating Disease Spread During Disaster Scenarios](https://direct.mit.edu/isal/proceedings/isal/35/123/116938)

**Shiyu Jiang**, Heejoong Kim, Fabio Henrique Tanaka, Claus Aranha, Anna Bogdanova, Kimia Ghobadi, Anton Dahbura. *The International Conference on Artificial Life*, 2023.

[Code](https://github.com/caranha/Koudou/tree/ALIFE_2023)

</div>
</div>

<div class='paper-box'><div class='paper-box-image'><div><div class="badge">Bioinformatics</div><img src='images/bioinformatics2022.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[HNOXPred: a web tool for the prediction of gas-sensing H-NOX proteins from amino acid sequence](https://academic.oup.com/bioinformatics/article/38/19/4643/6673135)

**Shiyu Jiang**, Hemn Barzan Abdalla, Chuyun Bi, Yi Zhu, Xuechen Tian, Yixin Yang, Aloysius Wong. *Bioinformatics*, 2022

[Website](https://www.hnoxpred.com/) | [Code](https://github.com/JasonJiangs/HNOX_Pred)
</div>
</div>

<div class='paper-box'><div class='paper-box-image'><div><div class="badge">IJCNN 2021</div><img src='images/ijcnn.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Deblur-yolo: Real-time object detection with efficient blind motion deblurring](https://ieeexplore.ieee.org/abstract/document/9534352)

Shen Zheng, Yuxiong Wu, **Shiyu Jiang**, Changjie Lu, Gaurav Gupta. *International Joint Conference on Neural Networks*, 2021

</div>
</div>

[comment]: <> (# üéñ Honors and Awards)

[comment]: <> (- *2021.10* Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ornare aliquet ipsum, ac tempus justo dapibus sit amet. )

[comment]: <> (- *2021.09* Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ornare aliquet ipsum, ac tempus justo dapibus sit amet. )

# üßë‚Äçüíª Research
- *2023.06 - Present*, Lab Specialist, [Chongzhi Zang Lab](https://zanglab.github.io/index.htm), Center for Public Health Genomics, University of Virginia, *Charlottesville, VA*.
- *2023.08 - Present*, Remote Research Assistant, [Xiaojie Qiu Lab](https://www.devo-evo.com/), Stanford School of Medicine, *Remote*.
- *2023.06 - 2023.12*, Remote Research Assistant, [Yanjun Li Lab](https://yanjun-li.com/index.html), University of Florida College of Pharmacy, *Remote*.
- *2022.09 - 2023.09*, Postgraduate Research Assistant, [Kimia Ghobadi Lab](https://systems.jhu.edu/kimia/), Johns Hopkins Withing School of Engineering, *Baltimore, MD*.
- *2021.08 - 2022.06*, Undergraduate Research Assistant, [Aloysius Wong Lab](https://csmt.wku.edu.cn/en/node/1777), Wenzhou-Kean University Department of Biology, *Wenzhou, China*.

# üßë‚Äçüíª Industry
- *2022.06 - 2022.08*, Software Engineer Intern, [Alibaba Cloud - PolarDB](https://www.alibabacloud.com/product/polardb), *Hangzhou, China*.
- *2021.09 - 2022.03*, Applied Research Intern, [Institute of Automation](https://people.ucas.edu.cn/~zhenshen?language=en), Chinese Academy of Sciences, *Beijing, China*.


# üî® Tools
## Model
- [Koudou](https://github.com/caranha/Koudou): 
  an agent-based model that simulates the infectious diseases.

[//]: # (- [REFEREE]&#40;https://github.com/OmicsML/scTab&#41; &#40;&#41;:)

[//]: # (  a federated foundation model for single-cell.)

[//]: # (- []&#40;&#41;:)

[//]: # (  a contrastive learning based model for RNA-small molecule binding affinity prediction.)
## Web server
- [HNOXPred](https://github.com/JasonJiangs/HNOX_Pred) (**Pred**iction of **H**eme-**N**itric oxide/**OX**ygen domains):
  a web server that predicts gas-sensing H-NOX proteins from amino acid sequences.

## Python Package
- [SICER 2.0](https://zanglab.github.io/SICER2/) (**S**patial-clustering **I**dentification of **C**hIP-**E**nriched **R**egions):
  a ChIP-Seq broad peak calling data analysis method.

# üåé Miscellaneous
Soccer (Visca Bar√ßa), Gym, Table tennis, Piano, Travel, etc.

<body>
  <a href="https://clustrmaps.com/site/1bt6x"  title="Visit tracker" >
    <img src="//www.clustrmaps.com/map_v2.png?d=aGpjzbKbHZT-5oLEhHvcK0igPnT7IvQmYxySQX6oPb4&cl=ffffff" />
  </a>
</body>