---
permalink: /
title: "Shiyu Jiang"
excerpt: ""
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

{% if site.google_scholar_stats_use_cdn %}
{% assign gsDataBaseUrl = "https://cdn.jsdelivr.net/gh/" | append: site.repository | append: "@" %}
{% else %}
{% assign gsDataBaseUrl = "https://raw.githubusercontent.com/" | append: site.repository | append: "/" %}
{% endif %}
{% assign url = gsDataBaseUrl | append: "google-scholar-stats/gs_data_shieldsio.json" %}

<span class='anchor' id='about-me'></span>

# üó® About Me
I am an incoming PhD student in Computational Biology and Bioinformatics (CBB) program at USC.
I am currently working as a visiting researcher at the Westlake University with 
[Prof. Fajie Yuan](https://en.westlake.edu.cn/faculty/fajie-yuan.html) and 
[Dr. Zitong Jerry Wang](https://cellethology.github.io/).
I received my Master's degree in Computer Science from Johns Hopkins University and 
Bachelor's degree in Computer Science from Wenzhou-Kean University.

Previously, I was advised by [Prof. Aloysius Wong](https://csmt.wku.edu.cn/en/node/1777) at Wenzhou-Kean University 
when I was an undergrad. After graduation, I was honored to work with 
[Prof. Chongzhi Zang](https://zanglab.github.io/index.htm) at the University of Virginia Medical School exploring 
computational genomics. In addition, I worked with [Prof. Yanjun Li](https://yanjun-li.com/index.html) at UFL on 
AI-driven drug discovery and later developed a single-cell foundation model in 
[Prof. Xiaojie Qiu's Lab](https://www.devo-evo.com/) at Stanford University. I am also collaborating with 
[Prof. Han Xiao's Group](https://xiao.rice.edu/) at Rice University Chemistry in developing a robust platform
for machine learning guided protein evolution.

I mainly work on the intersection of machine learning and biology in a broad scope. I am interested in foundation 
models for life sciences and how to use them to discover and interpret new biology. 

Research Interests:
- Deep Learning Methods for Molecular Interactions
- Large Language Model for Life Sciences
- Computational Genomics and Systems Biology
- Drug Discovery

# üìñ Educations
- *2025 - 2030*, incoming PhD student, Computational Biology and Bioinformatics. University of Southern California. *Los Angeles, CA*
- *2022 - 2024*, Master of Science in Engineering, Computer Science. Johns Hopkins University. *Baltimore, MD* 
- *2018 - 2022*, Bachelor of Science, Computer Science. Wenzhou-Kean University. *Wenzhou, China*

# üì∞ News
- *2025.06*: Our Sequence Display work is presented at [2025 Synthetic Biology: Engineering, Evolution, & Design](https://synbioconference.org/2025), see our [poster](https://jasonjiangs.github.io/images/seq_display_poster.png).
- *2025.06*: "[Predicting function of evolutionarily implausible DNA sequences](https://arxiv.org/abs/2506.10271)" was accepted by ICML 2025 Generative AI and Biology Workshop!
- *2025.04*: I will be joining the PhD program in Computational Biology and Bioinformatics at [USC QCB](https://www.qcb-dornsife.usc.edu/), Looking forward to the journey.
- *2025.01*: "[Toward a privacy-preserving predictive foundation model of single-cell transcriptomics with federated learning and tabular modeling](https://www.biorxiv.org/content/10.1101/2025.01.06.631427v1)" was released to bioRxiv, see our X [post](https://x.com/Xiaojie_Qiu/status/1876676759577661895).
- *2024.09*: one paper on accurate nanoplastics classification has been accepted by [ACS Nano](https://pubs.acs.org/doi/abs/10.1021/acsnano.4c08316), featured in the [cover](https://pubs.acs.org/cms/10.1021/ancac3.2024.18.issue-38/asset/ancac3.2024.18.issue-38.xlargecover-4.jpg).
- *2023.07*: one paper on developing a multi-agent simulation model for pandemic spread has been accepted by [ALIFE 2023](https://direct.mit.edu/isal/proceedings/isal2023/35/123/116938).
- *2022.08*: one paper on developing a protein recognition webserver has been accepted by [Bioinformatics](https://academic.oup.com/bioinformatics/article/38/19/4643/6673135).

# üìù Publications

[//]: # (<div class='paper-box'><div class='paper-box-image'><div><div class="badge">bioRxiv</div><img src='images/smartbind.png' alt="sym" width="75%"></div></div>)

[//]: # (<div class='paper-box-text' markdown="1">)

[//]: # ()
[//]: # ([SmartBind: Structure-Agnostic Ligand Discovery for RNA Targets Using Deep Learning]&#40;&#41;)

[//]: # (**Shiyu Jiang** ‚Ä†, Amirhossein Taghavi ‚Ä†, Tenghui Wang, Samantha M. Meyer, Jessica L. Childs-Disney, Chenglong Li, Mattew D. Disney, Yanjun Li. *bioRxiv*, 2025.)

[//]: # ([GitHub]&#40;https://github.com/AIDD-LiLab/SmartBind&#41;)

[//]: # (</div>)

[//]: # (</div>)

<div class='paper-box'><div class='paper-box-image'><div><div class="badge">ICML 2025 GenBio Workshop</div><img src='images/nullsettes.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Predicting function of evolutionarily implausible DNA sequences](https://arxiv.org/abs/2506.10271)
**Shiyu Jiang**, Xuyin Liu, Jerry Zitong Wang. *ICML 2025 Generative AI and Biology Workshop*, 2025.
[GitHub](https://github.com/cellethology/GLM-Nullsette-Benchmark) | [Poster](https://jasonjiangs.github.io/images/nullsettes_genbio2025_poster.png)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">bioRxiv</div><img src='images/qa.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Decoding the Molecular Language of Proteins with Evola](https://www.biorxiv.org/content/10.1101/2025.01.05.630192v1)
Xibin Zhou ‚Ä†, Chenchen Han ‚Ä†, Yingqi Zhang ‚Ä°, Jin Su ‚Ä°, Kai Zhuang ‚Ä°, **Shiyu Jiang** ‚Ä°, Zichen Yuan, Wei Zheng, Fengyuan Dai, Yuyang Zhou, Yuyang Tao, Dan Wu, Fajie Yuan. *bioRxiv*, 2025.
[Online Server](http://www.chat-protein.com/)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">bioRxiv</div><img src='images/tabula.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Toward a privacy-preserving predictive foundation model of single-cell transcriptomics with federated learning and tabular modeling](https://www.biorxiv.org/content/10.1101/2025.01.06.631427v1)
Jiayuan Ding ‚Ä†, Jianhui Lin ‚Ä†, **Shiyu Jiang** ‚Ä†, Yixin Wang, Ziyang Mao, Zhaoyu Fang, Jiliang Tang, Min Li, Xiaojie Qiu. *bioRxiv*, 2025.
[GitHub](https://github.com/aristoteleo/tabula)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">bioRxiv</div><img src='images/protrek.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[ProTrek: Navigating the Protein Universe through Tri-Modal Contrastive Learning](https://www.biorxiv.org/content/10.1101/2024.05.30.596740v2.abstract)
Jin Su ‚Ä†, Yan He ‚Ä†, Shiyang You ‚Ä†, **Shiyu Jiang** ‚Ä°, Xibin Zhou ‚Ä°, Xuting Zhang, Yuxuan Wang, Igor Tolstoy, Hongyuan Lu, Xing Chang, Fajie Yuan. *bioRxiv*, 2024, (In Submission).
[Online Server](http://search-protrek.com/)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">bioRxiv</div><img src='images/saprothub.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[SaprotHub: Making Protein Modeling Accessible to All Biologists](https://www.biorxiv.org/content/10.1101/2024.05.24.595648v5.abstract)
Jin Su, Zhikai Li, Chenchen Han, Yuyang Zhou, Yan He, Junjie Shan, Xibin Zhou, Xing Chang, **Shiyu Jiang**, Dacheng Ma, The OPMC, Martin Steinegger, Sergey Ovchinnikov, Fajie Yuan. *bioRxiv*, 2024, (In Submission). 
[GitHub](https://github.com/westlake-repl/SaprotHub?tab=readme-ov-file) | [OPMC](https://theopmc.github.io/)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">ACS Nano</div><img src='images/acsnano.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Integrating Metal‚ÄìPhenolic Networks-Mediated Separation and Machine Learning-Aided Surface-Enhanced Raman Spectroscopy for Accurate Nanoplastics Quantification and Classification](https://pubs.acs.org/doi/abs/10.1021/acsnano.4c08316)
Haoxin Ye, **Shiyu Jiang**, Yan Yan, Bin Zhao, Edward R Grant, David D Kitts, Rickey Y Yada, Anubhav Pratap-Singh, Alberto Baldelli, Tianxi Yang. *ACS Nano*, 2024.
[Featured on Cover](https://pubs.acs.org/cms/10.1021/ancac3.2024.18.issue-38/asset/ancac3.2024.18.issue-38.xlargecover-4.jpg)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">ALIFE 2023</div><img src='images/covid_sim.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Simulating Disease Spread During Disaster Scenarios](https://direct.mit.edu/isal/proceedings/isal/35/123/116938)
**Shiyu Jiang**, Heejoong Kim, Fabio Henrique Tanaka, Claus Aranha, Anna Bogdanova, Kimia Ghobadi, Anton Dahbura. *The International Conference on Artificial Life*, 2023.
[GitHub](https://github.com/caranha/Koudou/tree/ALIFE_2023)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">Bioinformatics</div><img src='images/bioinformatics2022.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[HNOXPred: a web tool for the prediction of gas-sensing H-NOX proteins from amino acid sequence](https://academic.oup.com/bioinformatics/article/38/19/4643/6673135)
**Shiyu Jiang**, Hemn Barzan Abdalla, Chuyun Bi, Yi Zhu, Xuechen Tian, Yixin Yang, Aloysius Wong. *Bioinformatics*, 2022.
[Online Server](https://www.hnoxpred.com/) | [GitHub](https://github.com/JasonJiangs/HNOX_Pred)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">IJCNN 2021</div><img src='images/ijcnn.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Deblur-yolo: Real-time object detection with efficient blind motion deblurring](https://ieeexplore.ieee.org/abstract/document/9534352)

Shen Zheng, Yuxiong Wu, **Shiyu Jiang**, Changjie Lu, Gaurav Gupta. *International Joint Conference on Neural Networks*, 2021

</div>
</div>

[comment]: <> (# üéñ Honors and Awards)

[comment]: <> (- *2021.10* Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ornare aliquet ipsum, ac tempus justo dapibus sit amet. )

[comment]: <> (- *2021.09* Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ornare aliquet ipsum, ac tempus justo dapibus sit amet. )

# üßë‚Äçüíª Experience
- *2024.08 - 2025.06*, Visiting Researcher, [Representation Learning Lab](https://github.com/westlake-repl) & [Cell Ethology Lab](https://cellethology.github.io/), Westlake University, *Hangzhou, China*.
- *2024.01 - 2024.07*, Lab Specialist, [Chongzhi Zang Lab](https://zanglab.github.io/index.htm), University of Virginia School of Medicine, *Charlottesville, VA*.
- *2022.06 - 2022.08*, Software Engineer Intern, [Alibaba Cloud - PolarDB](https://www.alibabacloud.com/product/polardb), *Hangzhou, China*.
- *2021.09 - 2022.03*, Applied Research Intern, [Institute of Automation](https://people.ucas.edu.cn/~zhenshen?language=en), Chinese Academy of Sciences, *Beijing, China*.


# üî® Models
## Genomics
- [Nullsettes](https://github.com/cellethology/GLM-Nullsette-Benchmark): 
  a synthetic biology benchmark simulating loss-of-function mutations via control element translocations, enabling zero-shot evaluation of genomic language models.

- [SICER 2.0](https://zanglab.github.io/SICER2/) (**S**patial-clustering **I**dentification of **C**hIP-**E**nriched **R**egions):
  a ChIP-Seq broad peak calling data analysis method.

- [Tabula](https://github.com/aristoteleo/tabula):
  A privacy-preserving predictive foundation model for single-cell transcriptomics, leveraging federated learning and tabular modeling.

## Protein
- [ProTrek](http://search-protrek.com/):
  a tri-modal protein language model that jointly models protein sequence, structure and function (SSF).

- [Evolla](http://www.chat-protein.com/):
  a protein-language generative model designed to decode the molecular language of proteins.

- [SaProtHub](https://github.com/westlake-repl/SaprotHub?tab=readme-ov-file):
  making Protein Modeling Accessible to All Biologists.

- [HNOXPred](https://www.hnoxpred.com/) (**Pred**iction of **H**eme-**N**itric oxide/**OX**ygen domains):
  a web server to predict gas-sensing H-NOX proteins from amino acid sequences.

## Molecule
- [SmartBind](https://github.com/AIDD-LiLab/SmartBind):
  SmartBind is a structure-free RNA-ligand interaction prediction method, which can be used for RNA-ligand virtual screening and binding site prediction.


## Other
- [Koudou](https://github.com/caranha/Koudou):
  an agent-based model that simulates the infectious disease spread under college town scenario.


# üåé Miscellaneous
Outside of work, you‚Äôll often find me at the gym, playing soccer, road cycling, or go hiking. 
I also enjoy playing table tennis and the piano occasionally.


<body>
  <a href="https://clustrmaps.com/site/1bt6x"  title="Visit tracker" >
    <img src="//www.clustrmaps.com/map_v2.png?d=aGpjzbKbHZT-5oLEhHvcK0igPnT7IvQmYxySQX6oPb4&cl=ffffff" />
  </a>
</body>