---
permalink: /
title: "Shiyu Jiang"
excerpt: ""
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

{% if site.google_scholar_stats_use_cdn %}
{% assign gsDataBaseUrl = "https://cdn.jsdelivr.net/gh/" | append: site.repository | append: "@" %}
{% else %}
{% assign gsDataBaseUrl = "https://raw.githubusercontent.com/" | append: site.repository | append: "/" %}
{% endif %}
{% assign url = gsDataBaseUrl | append: "google-scholar-stats/gs_data_shieldsio.json" %}

<span class='anchor' id='about-me'></span>

# üó® About Me
I am a first-year PhD student in the Computational Biology and Bioinformatics (CBB) program at the University of Southern California. 
My research lies at the intersection of AI and biology, where I design computational approaches to accelerate discoveries 
in synthetic biology, drug discovery, and molecular interaction.

Before starting my PhD, I was very fortunate to work with and learn from inspiring mentors and collaborators 
across these fields, you can find them in the experience panel.

# üìñ Educations
- *2025 - 2030*, PhD student, Computational Biology and Bioinformatics. University of Southern California. *Los Angeles, CA*
- *2022 - 2024*, Master of Science in Engineering, Computer Science. Johns Hopkins University. *Baltimore, MD* 
- *2018 - 2022*, Bachelor of Science, Computer Science. Wenzhou-Kean University. *Wenzhou, China*

# üì∞ News
- *2025.10*: "Engineering Unnatural Cells with a 21st Amino Acid as a Living Epigenetic Sensor" is accepted by ***[Nature Communications](https://www.nature.com/articles/s41467-025-64448-1)***!
- *2025.09*: "Small Molecule Approach to RNA Targeting Binder Discovery (SMARTBind) Using Deep Learning Without Structural Input" is released on [bioRxiv](https://www.biorxiv.org/content/10.1101/2025.09.24.678312v1).
- *2025.09*: "Tabula: A Tabular Self-Supervised Foundation Model for Single-Cell Transcriptomics" is accepted by ***[NeurIPS 2025](https://neurips.cc/virtual/2025/poster/117659)***!
- *2025.09*: "Biosynthesis of Unnatural Cyclodipeptides through Genetic Code Expansion and Cyclodipeptide Synthase Evolution" is accepted by ***[Journal of the American Chemical Society](https://doi.org/10.1021/jacs.5c08627)***!
- *2025.08*: "Evaluating DNA function understanding in genomic language models using evolutionarily implausible sequences" (follow-up work from the GenBio workshop) is released on [arXiv](https://arxiv.org/abs/2506.10271v3).
- *2025.08*: "SaprotHub: Democratizing Protein Language Model Training, Sharing and Collaboration for the Biology Community" is accepted by ***Nature Biotechnology***!
- *2025.07*: "A tri-modal protein language model enables advanced protein searches" is accepted by ***[Nature Biotechnology](https://www.nature.com/articles/s41587-025-02836-0)***!
- *2025.07*: "Predicting function of evolutionarily implausible DNA sequences" is presented at [Q-BIO 2025 Conference: Emergent Orders in Living Systems Across Scales](https://cqb.pku.edu.cn/info/1064/3011.htm), see our [poster](https://jasonjiangs.github.io/images/nullsettes_poster.png).
- *2025.06*: "Sequence Display-Enabled Machine Learning for Protein Evolution" is presented at [2025 Synthetic Biology: Engineering, Evolution, & Design](https://aiche.confex.com/aiche/seed2025/meetingapp.cgi/Paper/708439), see our [poster](https://jasonjiangs.github.io/images/seq_display_poster.png).
- *2025.06*: "Predicting function of evolutionarily implausible DNA sequences" is accepted by ***[ICML 2025 Generative AI and Biology Workshop](https://openreview.net/forum?id=YEmMJHZSGv)***!
- *2025.04*: I will be joining the PhD program in Computational Biology and Bioinformatics at [USC QCB](https://www.qcb-dornsife.usc.edu/), looking forward to the journey.
- *2025.01*: "Toward a privacy-preserving predictive foundation model of single-cell transcriptomics with federated learning and tabular modeling" is released on [bioRxiv](https://www.biorxiv.org/content/10.1101/2025.01.06.631427v1), see our [post](https://x.com/Xiaojie_Qiu/status/1876676759577661895).
- *2024.09*: one paper on accurate nanoplastics classification has been accepted by ***ACS Nano***([paper link](https://pubs.acs.org/doi/abs/10.1021/acsnano.4c08316)), featured in the [cover](https://pubs.acs.org/cms/10.1021/ancac3.2024.18.issue-38/asset/ancac3.2024.18.issue-38.xlargecover-4.jpg).
- *2023.07*: one paper on developing a multi-agent simulation model for pandemic spread is accepted by ***ALIFE 2023*** ([paper link](https://direct.mit.edu/isal/proceedings/isal2023/35/123/116938)).
- *2022.08*: one paper on developing a protein recognition webserver is accepted by ***Bioinformatics*** ([paper link](https://academic.oup.com/bioinformatics/article/38/19/4643/6673135)).

# üìù Publications
<div class='paper-box'><div class='paper-box-image'><div><div class="badge">bioRxiv</div><img src='images/smartbind.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Small Molecule Approach to RNA Targeting Binder Discovery (SMARTBind) Using Deep Learning Without Structural Input](https://www.biorxiv.org/content/10.1101/2025.09.24.678312v1)

**Shiyu Jiang** ‚Ä†, Amirhossein Taghavi ‚Ä†, Tenghui Wang, Samantha M. Meyer, Jessica L. Childs-Disney, Chenglong Li, Mattew D. Disney, Yanjun Li. *bioRxiv*, 2025. (In Submission)

[GitHub](https://github.com/AIDD-LiLab/SMARTBind)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">arXiv</div><img src='images/nullsettes_v2.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Evaluating DNA function understanding in genomic language models using evolutionarily implausible sequences](https://arxiv.org/abs/2506.10271v3)

**Shiyu Jiang**, Xuyin Liu, Jerry Zitong Wang. *arXiv*, 2025. (In Submission)

[GitHub](https://github.com/cellethology/GLM-Nullsette-Benchmark)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">bioRxiv</div><img src='images/sequence display.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Sequence Display: Generating Large-Scale Sequence‚ÄìActivity Datasets to Advance Universal Protein Evolution]()

Linqi Cheng ‚Ä†, Xinzhe Zheng ‚Ä†, **Shiyu Jiang** ‚Ä†, Hu Y, Liu Y, Yang K, Rui J, Ding H, Zhang M, Yuan T, Ye H, Li C, Kevin K. Yang, Xiongyi Huang, Han Xiao. *bioRxiv*, 2025. (In Submission)

[GitHub](https://github.com/SophieSarceau/SequenceDisplay-ML)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">bioRxiv</div><img src='images/qa.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Decoding the Molecular Language of Proteins with Evola](https://www.biorxiv.org/content/10.1101/2025.01.05.630192v1)

Xibin Zhou ‚Ä†, Chenchen Han ‚Ä†, Yingqi Zhang ‚Ä°, Jin Su ‚Ä°, Kai Zhuang ‚Ä°, **Shiyu Jiang** ‚Ä°, Zichen Yuan, Wei Zheng, Fengyuan Dai, Yuyang Zhou, Yuyang Tao, Dan Wu, Fajie Yuan. *bioRxiv*, 2025.

[Online Server](http://www.chat-protein.com/)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">NeurIPS 2025</div><img src='images/tabula_nips.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Tabula: A Tabular Self-Supervised Foundation Model for Single-Cell Transcriptomics](https://neurips.cc/virtual/2025/poster/117659)

Jiayuan Ding ‚Ä†, Jianhui Lin ‚Ä†, **Shiyu Jiang** ‚Ä†, Yixin Wang, Ziyang Mao, Zhaoyu Fang, Jiliang Tang, Min Li, Xiaojie Qiu. *NeurIPS* (poster), 2025.

[GitHub](https://github.com/aristoteleo/tabula)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">Nature Communications</div><img src='images/NC_yu.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Engineering Unnatural Cells with a 21st Amino Acid as a Living Epigenetic Sensor](https://www.nature.com/articles/s41467-025-64448-1)

Hu Y, Wang Y, Cheng L, Wang C, Liu Y, Wang Y, Chen Y, Yang S, Guo Y, **Jiang S**, Yang K, Xiao H. *Nature Communications*, 2025.

</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">JACS</div><img src='images/jacs_1.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Biosynthesis of Unnatural Cyclodipeptides through Genetic Code Expansion and Cyclodipeptide Synthase Evolution](https://pubs.acs.org/doi/10.1021/jacs.5c08627)

Hu Y ‚Ä†, Cheng L ‚Ä†, Liu Y, Liu R, **Jiang S**, Yuan T, Wang Y, Ye H, Xiao H. *Journal of the American Chemical Society*, 2025.

[GitHub](https://github.com/linqicheng-xiao/gmx_mmpbsa_py)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">Nature Biotechnology</div><img src='images/protrek.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[A tri-modal protein language model enables advanced protein searches](https://www.nature.com/articles/s41587-025-02836-0)

Jin Su ‚Ä†, Yan He ‚Ä†, Shiyang You ‚Ä†, **Shiyu Jiang**, Xibin Zhou, Xuting Zhang, Yuxuan Wang, Xining Su, Igor Tolstoy, Xing Chang, Hongyuan Lu, Fajie Yuan. *Nature Biotechnology*, 2025.

[Online Server](http://search-protrek.com/)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">Nature Biotechnology</div><img src='images/saprothub.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[SaprotHub: Democratizing Protein Language Model Training, Sharing and Collaboration for the Biology Community](https://www.biorxiv.org/content/10.1101/2024.05.24.595648v5.abstract)

Jin Su, Zhikai Li, Chenchen Han, Yuyang Zhou, Yan He, Junjie Shan, Xibin Zhou, Xing Chang, **Shiyu Jiang**, Dacheng Ma, The OPMC, Martin Steinegger, Sergey Ovchinnikov, Fajie Yuan. *Nature Biotechnology*, 2025. (Accepted)

[GitHub](https://github.com/westlake-repl/SaprotHub?tab=readme-ov-file) | [OPMC](https://theopmc.github.io/)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">ICML 2025 GenBio Workshop</div><img src='images/nullsettes.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Predicting function of evolutionarily implausible DNA sequences](https://openreview.net/forum?id=YEmMJHZSGv)

**Shiyu Jiang**, Xuyin Liu, Jerry Zitong Wang. *ICML 2025 Generative AI and Biology Workshop*, 2025.

[GitHub](https://github.com/cellethology/GLM-Nullsette-Benchmark)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">ACS Nano</div><img src='images/acsnano.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Integrating Metal‚ÄìPhenolic Networks-Mediated Separation and Machine Learning-Aided Surface-Enhanced Raman Spectroscopy for Accurate Nanoplastics Quantification and Classification](https://pubs.acs.org/doi/abs/10.1021/acsnano.4c08316)

Haoxin Ye, **Shiyu Jiang**, Yan Yan, Bin Zhao, Edward R Grant, David D Kitts, Rickey Y Yada, Anubhav Pratap-Singh, Alberto Baldelli, Tianxi Yang. *ACS Nano*, 2024.

[Featured on Cover](https://pubs.acs.org/cms/10.1021/ancac3.2024.18.issue-38/asset/ancac3.2024.18.issue-38.xlargecover-4.jpg)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">ALIFE 2023</div><img src='images/covid_sim.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Simulating Disease Spread During Disaster Scenarios](https://direct.mit.edu/isal/proceedings/isal/35/123/116938)

**Shiyu Jiang**, Heejoong Kim, Fabio Henrique Tanaka, Claus Aranha, Anna Bogdanova, Kimia Ghobadi, Anton Dahbura. *The International Conference on Artificial Life*, 2023.

[GitHub](https://github.com/caranha/Koudou/tree/ALIFE_2023)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">Bioinformatics</div><img src='images/bioinformatics2022.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[HNOXPred: a web tool for the prediction of gas-sensing H-NOX proteins from amino acid sequence](https://academic.oup.com/bioinformatics/article/38/19/4643/6673135)

**Shiyu Jiang**, Hemn Barzan Abdalla, Chuyun Bi, Yi Zhu, Xuechen Tian, Yixin Yang, Aloysius Wong. *Bioinformatics*, 2022.

[Online Server](https://www.hnoxpred.com/) | [GitHub](https://github.com/JasonJiangs/HNOX_Pred)
</div>
</div>


<div class='paper-box'><div class='paper-box-image'><div><div class="badge">IJCNN 2021</div><img src='images/ijcnn.png' alt="sym" width="75%"></div></div>
<div class='paper-box-text' markdown="1">

[Deblur-yolo: Real-time object detection with efficient blind motion deblurring](https://ieeexplore.ieee.org/abstract/document/9534352)

Shen Zheng, Yuxiong Wu, **Shiyu Jiang**, Changjie Lu, Gaurav Gupta. *International Joint Conference on Neural Networks*, 2021

</div>
</div>

[comment]: <> (# üéñ Honors and Awards)

[comment]: <> (- *2021.10* Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ornare aliquet ipsum, ac tempus justo dapibus sit amet. )

[comment]: <> (- *2021.09* Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ornare aliquet ipsum, ac tempus justo dapibus sit amet. )

# üßë‚Äçüíª Experience
<ul class="education-timeline">

    <li class="timeline-item-edu">
        <div class="timeline-icon">
            <img src="images/USC_logo.png" alt="USC Logo">
        </div>
        <div class="timeline-content-edu">
            <p class="date">2025.09 - Present</p>
            <h4 class="title">Graduate Research Assistant</h4>
            <p class="school"><a href="https://www.qcb-dornsife.usc.edu/" target="_blank">University of Southern California</a>, Department of Quantitative and Computational Biology</p>
        </div>
    </li>

    <li class="timeline-item-edu">
        <div class="timeline-icon">
            <img src="images/WLU_logo.png" alt="WLU Logo">
        </div>
        <div class="timeline-content-edu">
            <p class="date">2024.08 - 2025.06</p>
            <h4 class="title">Research Associate</h4>
            <p class="date">Development and evaluation of protein/genomic language model | Advisor: <a href="https://fajieyuan.github.io/" target="_blank">Prof. Fajie Yuan</a> & <a href="https://www.cellethology.org/" target="_blank">Dr. Zitong Jerry Wang</a></p>
            <p class="school"><a href="https://en.westlake.edu.cn/" target="_blank">Westlake University</a>, School of Engineering & Center for Interdisciplinary Studies, School of Science</p>
        </div>
    </li>

    <li class="timeline-item-edu">
        <div class="timeline-icon">
            <img src="images/rice_logo.png" alt="RiceU Logo">
        </div>
        <div class="timeline-content-edu">
            <p class="date">2023 - 2025</p>
            <h4 class="title">Remote Research Assistant</h4>
            <p class="date">Machine learning for protein evalution | Advisor: <a href="https://xiao.rice.edu/" target="_blank">Prof. Han Xiao</a></p>
            <p class="school"><a href="https://chemistry.rice.edu/" target="_blank">Rice University</a>, Department of Chemistry</p>
        </div>
    </li>

    <li class="timeline-item-edu">
        <div class="timeline-icon">
            <img src="images/UF_logo.png" alt="UF Logo">
        </div>
        <div class="timeline-content-edu">
            <p class="date">2023 - 2025</p>
            <h4 class="title">Remote Research Assistant</h4>
            <p class="date">RNA-small molecule drug discovery and molecule generation | Advisor: <a href="https://yanjun-li.com/" target="_blank">Prof. Yanjun Li</a> & <a href="https://disney.scripps.ufl.edu/" target="_blank">Prof. Matthew D. Disney</a></p>
            <p class="school"><a href="https://pharmacy.ufl.edu/" target="_blank">University of Florida</a>, College of Pharmacy & <a href="https://wertheim.scripps.ufl.edu/" target="_blank">UF Scripps Institute</a>, Department of Chemistry</p>
        </div>
    </li>

    <li class="timeline-item-edu">
        <div class="timeline-icon">
            <img src="images/STF_logo.png" alt="Stanford Logo">
        </div>
        <div class="timeline-content-edu">
            <p class="date">2023 - 2025</p>
            <h4 class="title">Remote Research Assistant</h4>
            <p class="date">Foundation model for single-cell transcriptomics | Advisor: <a href="https://www.devo-evo.com/" target="_blank">Prof. Xiaojie Qiu</a></p>
            <p class="school"><a href="https://med.stanford.edu/genetics.html?tab=proxy" target="_blank">Stanford University</a>, Department of Genetics</p>
        </div>
    </li>

    <li class="timeline-item-edu">
        <div class="timeline-icon">
            <img src="images/UVA_logo.png" alt="UVA Logo">
        </div>
        <div class="timeline-content-edu">
            <p class="date">2024.01 - 2024.07</p>
            <h4 class="title">Lab Specialist</h4>
            <p class="date">ChIP-Seq peak calling tool | Advisor: <a href="https://zanglab.github.io/index.htm" target="_blank">Prof. Chongzhi Zang</a></p>
            <p class="school"><a href="https://med.virginia.edu/genome-sciences/" target="_blank">University of Virginia</a>, Department of Genome Sciences</p>
        </div>
    </li>

    <li class="timeline-item-edu">
        <div class="timeline-icon">
            <img src="images/AlibabaCloud_logo.png" alt="Alibaba Cloud Logo">
        </div>
        <div class="timeline-content-edu">
            <p class="date">2022.06 - 2022.08</p>
            <h4 class="title">Software Engineer Intern</h4>
            <p class="school"><a href="https://www.alibabacloud.com/en?_p_lc=5" target="_blank">Alibaba Cloud</a> </p>
        </div>
    </li>
</ul>

# üî® Models and Tools
## Genomics
- [Nullsettes](https://github.com/cellethology/GLM-Nullsette-Benchmark): 
  a synthetic biology benchmark simulating loss-of-function mutations via control element translocations, enabling zero-shot evaluation of genomic language models.

- [SICER 2.0](https://zanglab.github.io/SICER2/) & [Clipper dev Version](https://github.com/zanglab/SICER2/tree/sicer_dev) (**S**patial-clustering **I**dentification of **C**hIP-**E**nriched **R**egions):
  a redesigned ChIP-Seq broad peak calling data analysis method.

- [Tabula](https://github.com/aristoteleo/tabula):
  A privacy-preserving predictive foundation model for single-cell transcriptomics, leveraging federated learning and tabular modeling.

## Protein
- [Sequence display](https://github.com/SophieSarceau/SequenceDisplay-ML):
  a platform that integrates large‚Äëscale sequence‚Äìactivity datasets with protein language models to map activity landscapes and identify high‚Äëperformance protein variants.

- [ProTrek](http://search-protrek.com/):
  a tri-modal protein language model that jointly models protein sequence, structure and function (SSF).

- [Evolla](http://www.chat-protein.com/):
  a protein-language generative model (Protein ChatGPT) designed to decode the molecular language of proteins.

- [SaProtHub](https://github.com/westlake-repl/SaprotHub?tab=readme-ov-file):
  making Protein Modeling Accessible to All Biologists.

- [HNOXPred](https://www.hnoxpred.com/) (**Pred**iction of **H**eme-**N**itric oxide/**OX**ygen domains):
  a web server to predict gas-sensing H-NOX proteins from amino acid sequences.

## Molecular interacion
- [SMARTBind](https://github.com/AIDD-LiLab/SMARTBind):
  SMARTBind is a structure-agnostic RNA-ligand interaction prediction method, which can be used for RNA-ligand virtual screening and binding site prediction.

- [gmx_mmpbsa_py](https://github.com/linqicheng-xiao/gmx_mmpbsa_py):
  an easy-to-use Python script that integrates GROMACS molecular dynamics trajectories with APBS to compute protein‚Äìligand binding free energies using the MM/PBSA method.

## Other
- [Koudou](https://github.com/caranha/Koudou):
  an agent-based model that simulates the infectious disease spread under college town scenario.


# üåé Miscellaneous
Outside of work, you‚Äôll often find me at the gym, playing soccer, road cycling, or go hiking. 
I also enjoy playing table tennis and the piano occasionally.


<body>
  <a href="https://clustrmaps.com/site/1bt6x"  title="Visit tracker" >
    <img src="//www.clustrmaps.com/map_v2.png?d=aGpjzbKbHZT-5oLEhHvcK0igPnT7IvQmYxySQX6oPb4&cl=ffffff" />
  </a>
</body>